korman/properties/modifiers/gui.py

```diff
@@ -330,3 +331,249 @@ def _create_moul_nodes(self, clickable_object, nodes, journalnode, age_name):
+                               description="Height scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    # TODO: link image
+    # TODO: stamp texture
+    stamp_x = IntProperty(name="Stamp Position X",
+                          description="Width scale",
+                          default=100, min=0, max=100,
+                          subtype="PERCENTAGE")
+    stamp_y = IntProperty(name="Stamp Position Y",
+                          description="Height scale",
+                          default=100, min=0, max=100,
+                          subtype="PERCENTAGE")
+
+    def export(self, exporter, bo, so):
+        our_versions = (globals()[j] for j in self.versions)
+        version = exporter.mgr.getVer()
+        if version not in our_versions:
+            # We aren't needed here
+            exporter.report.port("Object '{}' has a LinkingBookMod not enabled for export to the selected engine.  Skipping.",
+                                 bo.name, version, indent=2)
+            return
+
+
+        if self.clickable is None:
+            raise ExportError("{}: Linking Book modifier requires a clickable!".format(bo.name))
+
+        if self.seek_point is None:
+            raise ExportError("{}: Linking Book modifier requires a seek point!".format(bo.name))
+
+        # Auto-generate a six-foot cube region around the clickable if none was provided.
+        if self.clickable_region is None:
+            # Create a region for the clickable's condition
+            rgn_mesh = bpy.data.meshes.new("{}_LinkingBook_ClkRgn".format(self.key_name))
+            self.temp_rgn = bpy.data.objects.new("{}_LinkingBook_ClkRgn".format(self.key_name), rgn_mesh)
+            bm = bmesh.new()
+            bmesh.ops.create_cube(bm, size=(6.0))
+            bmesh.ops.transform(bm, matrix=mathutils.Matrix.Translation(self.clickable.location - self.temp_rgn.location), space=self.temp_rgn.matrix_world, verts=bm.verts)
+            bm.to_mesh(rgn_mesh)
+            bm.free()
+
+            # No need to enable the object as a Plasma object; we're exported automatically as part of the node tree.
+            # It does need a page, however, so we'll put it in the same place as the linking book object itself.
+            self.temp_rgn.plasma_object.page = self.clickable.plasma_object.page
+            bpy.context.scene.objects.link(self.temp_rgn)
+        else:
+            self.temp_rgn = self.clickable_region
+
+        # Generate the logic nodes
+        with self.generate_logic(bo, age_name=exporter.age_name, version=version) as tree:
+            tree.export(exporter, bo, so)
+
+        # Get rid of our temporary clickable region
+        if self.clickable_region is None:
+            bpy.context.scene.objects.unlink(self.temp_rgn)
+
+        # Get rid of our temporary seek point
+        if self.clickable_region is None:
+            bpy.context.scene.objects.unlink(self.temp_seek)
```

Comment:
If the exporter crashes, these objects are left orphaned. See `utils.TemporaryObject`.

Reply:
Oh, yeah!  Forgot you added that!